src/config/dynamic-theme-fixes.config

```diff
@@ -157,6 +157,24 @@ body > #root > #wrapper > div[class^='c'] > #main-content > div:first-child
 
 ================================
 
+*.vultr.com
+
+CSS
+.svg-banner-shape {
+    fill: var(--darkreader-neutral-background);
```

Comment:
don't forget the `!important;`

Reply:
Hmm, maybe it is a good idea to tell the auto-formatter to do this too.

Reply:
> maybe it is a good idea to tell the auto-formatter to do this too.

Not really, certain fixes will break when you add `!important;` and it's hard to migrate this false-positives